groups.php

```diff
@@ -78,8 +78,8 @@
 
 <script src="scripts/vendor/bootstrap-select.min.js"></script>
 <script src="scripts/vendor/bootstrap-toggle.min.js"></script>
```

Comment:
I won't add a comment on all of them, you get the idea, I think!

Unless you missed these on purpose? Would be happy to hear your thoughts